docs/source/intro/install_conda.rst

```diff
@@ -84,6 +84,11 @@ This is the recommended method for users on shared systems and systems where the
        Last name: name
        Institution: where-i-work
 
+   .. admonition:: \ \
+       :class: attention title-icon-troubleshoot
```

Comment:
Ok, sorry, I did not intend to motivate the introduction of some work-around, I just did not know whether this kind of admonition requires a title or not. Let's stick with the standard style with title then.